### PR TITLE
Add browse_dataset for visualization

### PR DESCRIPTION
**Pull request by canoe-Z**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Fix the bug in imshow_det_rbboxes when input bboxes are without scores. Add browse_dataset for visualization.

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by yangxue0827** [comment]
Thanks for your contribution. Detailed docstring should be added to all functions, you can refer to other scripts.

**Comment by canoe-Z** [comment]
> Thanks for your contribution. Detailed docstring should be added to all functions, you can refer to other scripts.

Docstring has been added.

**Review by zytx121 — APPROVED** [review]
LGTM

File: mmrotate/core/visualization/image.py [hunk 1]
```diff
@@ -63,17 +63,20 @@ def imshow_det_rbboxes(img,
     assert labels.ndim == 1
     img = imread(img)
 
-    scores = bboxes[:, -1]
-    inds = scores > score_thr
-    bboxes = bboxes[inds, :]
-    labels = labels[inds]
+    if score_thr > 0:
+        assert bboxes is not None and bboxes.shape[1] == 6
```
Comment (jbwang1997) [inline mmrotate/core/visualization/image.py:67]: The assertion needs to be put out of the if branch. We could put it after `assert bboxes.ndim == 2`

**Review by jbwang1997 — APPROVED** [review]
LGTM

**Pull request merged**